docs/src/inverse_problems/optimization_ode_param_fitting.md

```diff
@@ -24,7 +28,7 @@ ps_true = [:kB => 1.0, :kD => 0.1, :kP => 0.5]
 using OrdinaryDiffEqDefault
 oprob_true = ODEProblem(rn, u0, (0.0, 10.0), ps_true)
 true_sol = solve(oprob_true)
-data_sol = solve(oprob_true; saveat=1.0)
+data_sol = solve(oprob_true; saveat = 1.0)
```

Comment:
I think when I wrote these I didn't use to put spaces around `=` in kwargs, but that should be changed to conform to standard. At some point I will have a look at more tutorials to see if there are other places I don't use the SciML standard.